.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Report any issues found and help us improve
+title: 'fix: [ISSUE]'
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
+A clear and concise description of what the bug is.
+
+## To reproduce
+Steps to reproduce the behavior.
+
+## Expected behavior
+A clear and concise description of what you expected to happen.
+
+## Environment details
+ - Version: [e.g. 0.41.2]
```

Comment:
We need to display the app version more clearly to the user - preferably at both the FE and BE while running the images

Reply:
@hari-kuriakose @chandrasekharan-zipstack we could consider adding this in to the run script as a start.

Reply:
@chandrasekharan-zipstack @ritwik-g Couldn't agree more.